tests/pl/Tests_2004/Markdown Documentation - Basics.html

```diff
@@ -114,7 +114,7 @@ <h2>Lists</h2>
 
 <p>Unordered (bulleted) lists use asterisks, pluses, and hyphens (<code>*</code>,
 <code>+</code>, and <code>-</code>) as list markers. These three markers are
-interchangable; this:</p>
+interchangeable; this:</p>
```

Comment:
The `pl` test files are borrowed directly from the Perl Markdown project and should always remain unchanged. Please do not make any changes to these files. These changes need to be undone before I can accept this.